src/entrypoints/exception-autocapture.ts

```diff
@@ -46,9 +46,36 @@ const wrapUnhandledRejection = (captureFn: (props: Properties) => void) => {
     }
 }
 
+const wrapConsoleError = (captureFn: (props: Properties) => void) => {
+    const con = console as any
+    if (!con) {
+        logger.info('window not available, cannot wrap onUnhandledRejection')
+    }
+
+    const originalConsoleError = con.error
+
+    con.error = function (...args: any[]): boolean {
+        const event = args.join(' ')
+        const error = args.find((arg) => arg instanceof Error)
+        const errorProperties = error
+            ? errorToProperties({ event, error })
+            : errorToProperties({ event }, { syntheticException: new Error('PostHog syntheticException') })
+
+        captureFn(errorProperties)
```

Comment:
might be cool to indicate if replay and console capture is running
to help with linking between the two

but i guess we have 99% of that anyway

Reply:
I did try to consolidate the two but the replay one looks to be within rrweb :/